tdms/tests/unit/matlab_benchmark_scripts/benchmark_test_field_interpolation_H.m

```diff
@@ -0,0 +1,193 @@
+%{
+        BENCHMARK FOR H-FIELD INTERPOLATION METHODS
+
+Tests the interpolation accuracy for the H-field components, when
+interpolated in each direction to the centre of a Yee cell - benchmarking
+test_field_interpolation::TEST_CASE("H-field interpolation check").
+
+The H-field used will have identical field components, each of which are
+given by the expression:
+
+H{x,y,z}(xx,yy,zz) = cos(0.5\pi zz) * exp(-yy^2) * ( 1./ (5xx^2+1) ),
+
```

Comment:
See comment above about alternative function. Also, has 0.5\pi been chosen intentionally? This is sub-optimal as (if I understand correctly), we have at least 16 points per sinusoidal oscillation, which is too many to warrant band-limited interpolation

Reply:
$0.5\pi$ is me incorrectly translating my paper notes -> code. I've also taken a similar form for the $H$-field as for the $E$-field you suggested above. The tests have been updated with the new functional forms.